modules/http/src/main/java/com/spotify/ffwd/http/HttpClient.java

```diff
@@ -36,16 +35,23 @@
 import okhttp3.RequestBody;
 import okhttp3.Response;
 
-@Data
 public class HttpClient {
-    public static final String V1_BATCH_ENDPOINT = "v1/batch";
-    public static final String PING_ENDPOINT = "ping";
+    private static final String V1_BATCH_ENDPOINT = "v1/batch";
+    private static final String PING_ENDPOINT = "ping";
 
     private final AsyncFramework async;
     private final ObjectMapper mapper;
     private final OkHttpClient httpClient;
     private final String baseUrl;
 
+    public HttpClient(AsyncFramework async, ObjectMapper mapper, OkHttpClient httpClient,
```

Comment:
I think you'll need to add jackson annotations on the constructor. That's something the lombok `Data` classes do invisibly.

Reply:
:+1: